libcudacxx/examples/rtc_example.cpp

```diff
@@ -22,22 +22,25 @@ THE SOFTWARE.
 
 */
 
-#include <nvrtc.h>
 #include <cuda.h>
```

Comment:
This rule in clang-format means that `<cuda.h>` is treated more like `<cuda/std/...>` and not like `<nvrtc.h>`.

https://github.com/NVIDIA/cccl/blob/bff5288bac1989f888d2014f8930adf8c308c446/.clang-format#L116-L118


You probably want to change the clang-format rules to group libcudacxx separately from `<cuda.h>` and `<cuda_runtime.h>`. Maybe change the rule to match `^<cuda[a-z_/]*>$`?

Reply:
We probably want a similar change for CUB/Thrust, to avoid conflicts with other CUDA libraries (e.g. `cublas` which has prefix `cub`). It'd be good to play with this a little bit.
- `^<cub/[a-z_/.]*>$`
- `^<thrust/[a-z_/.]*>$`

Reply:
I added the slash at the end and also two more categories, one our config file that need to come first and one the cuda related files